conbench/templates/run.html

```diff
@@ -66,6 +68,12 @@
 
 $(document).ready(function() {
     $('#unit-tooltip').tooltip()
+    $("#run-form").find("#delete").attr("type", "button");
+    $("#run-form").find("#delete").attr("data-toggle", "modal");
+    $("#run-form").find("#delete").attr("data-target", "#confirm-delete");
+    $("#run-form").find("#delete").attr("data-form-id", "#run-form");
+    $("#run-form").find("#delete").attr("data-href", "{{ url_for('app.run', run_id=benchmarks[0].run_id) }}");
```

Comment:
I know it's odd, but you can't assume a benchmark has runs.

If a benchmark doesn't have runs, you'll get an error like this:

`jinja2.exceptions.UndefinedError: list object has no element 0`

You should be able to wrap in in something like this:

```
        {% if benchmarks %}
        ...
        {% endif %}
```

Better, perhaps pass the run_id to the HTML via the controller, otherwise we won't be able to delete a run that has zero benchmarks.

``` 
        return self.render_template(
            "run.html",
            application=Config.APPLICATION_NAME,
            title="Run",
            benchmarks=benchmarks,
            compare_runs_url=compare_runs_url,
            form=form,
            run_id=contender_run["id"],
        )
```

Reply:
Actually, you won't necessarily have a populated `contender_run` at that point, so instead you'll need to pass `run_id` from:

```
    def get(self, run_id):
        if self.public_data_off():
            return self.redirect("app.login")
```

into:

```
    def page(self, benchmarks, baseline_run, contender_run, form):
        compare_runs_url = None
```

And then:

```
        return self.render_template(
            "run.html",
            application=Config.APPLICATION_NAME,
            title="Run",
            benchmarks=benchmarks,
            compare_runs_url=compare_runs_url,
            form=form,
            run_id=run_id,
        )
```

Or something like that.